packages/kbn-test/src/kbn_client/kbn_client_version.ts

```diff
@@ -19,6 +19,13 @@ export class KbnClientVersion {
     }
 
     const status = await this.status.get();
+
+    if (!status.version) {
```

Comment:
I was receiving this error but could not see the response body to debug the issue:

```
Cannot read properties of undefined (reading 'number')
```

Reply:
Do you have an example of the stringified status response?

Reply:
No sorry thats why I added this change so hopefully we can catch it next time, it's strange that we get _some_ JSON back

Reply:
found at https://github.com/elastic/kibana/blob/77be3c0b327da34587930125de689a0bcce459b9/packages/kbn-test/src/kbn_client/kbn_client_status.ts#L19-L34.  lgtm